services/core-api/app/api/mines/explosives_permit/resources/explosives_permit.py

```diff
@@ -221,8 +220,7 @@ def put(self, mine_guid, explosives_permit_guid):
             explosives_permit.save()
         except Exception as e:
             current_app.logger.error(e)
-            raise MineException("Oops!, Something went wrong while updaitng the explosive permit information",
-                                detailed_error = e)
+            raise MineException("Something went wrong while updaitng the explosive permit information")
```

Comment:
typo on updating.
I do wonder a bit about using the "mine exception" type for all these- I do see that we also have `ExplosivesPermitDocumentException`, `ExplosivesPermitExeption` (some spelling errors there 😓 ) but also not sure that we need a different exception type for every object type when we're setting a custom error message in every instance.

Reply:
Removed unnecessary exceptions and used only `MineException` with appropriate message and status codes.

Reply:
I know the typo is pre-existing, but should still change 'updaitng' to 'updating'

Reply:
Well, that's updated already. Please see all the changes.

Reply:
So it is! Must have been looking at it wrong. 🤷‍♀️